src/main/java/org/openrewrite/github/AddCronTrigger.java

```diff
@@ -58,4 +90,57 @@ public String getDescription() {
     public Duration getEstimatedEffortPerOccurrence() {
         return Duration.ofMinutes(5);
     }
+
+    static class RandomCronExpression {
+        private final Random random;
+        public RandomCronExpression(Random random){
+            this.random = random;
+        }
+
+        public int random(int min, int max) {
+            return random.nextInt(max + 1 - min) + min;
+        }
+
+        public int minute(){
+            return random(0,59);
+        }
+
+        public int hour(){
+            return random(0,12);
+        }
+
+        public int randomDayOfTheWeek() {
+            return random(0,6);
+        }
+
+        public String dailyCron(){
+            return String.format("%d %d * * *",minute(),hour());
+        }
+
+        public String weeklyCron(){
+            return String.format("%d %d * * %s",minute(),hour(), randomDayOfTheWeek());
+        }
+
+        public String monthlyCron(){
+            return String.format("%d %d %s * %s",minute(),hour(), dayOfTheMonth(), randomDayOfTheWeek());
```

Comment:
Does this really need a day off the week?
I'm wondering if it will only schedule if that matches up to the day off the month, so once in seven times. But it's quite early here so mostly a question, as I'm not behind a laptop yet.

Reply:
I am unsure how cron will behave here, to be honest

I found online that it would manage that and schedule it on a different day if needed. But to avoid confusions I removed it with https://github.com/openrewrite/rewrite-github-actions/pull/56/commits/42edd2c15075e1214d1f3f7a7a41bf42fe7f73dd